class ExceptionPiezaYaExiste(Exception):
    def __init__(self):
        pass 